src/components/AddUserForm/AddUserForm.js

```diff
@@ -0,0 +1,147 @@
+            })
+        } else {
+        // Todo: добавить здесь проверку, если пользователь НЕ ВВЕЛ в инпуты ничего - не создаем пост
+        //  и обновляем currentError в стейте, кладем сообщение, что пошло не так
+        //  если пользователь ввел данные - создаем постзадачу
+
+        const {addUser} = this.props;
+
+        const user = {
+            name: this.userNameRef.current.value,
+            lastName: this.userLastNameRef.current.value,
+            id: uniqId()
+        };
+
+        addUser(user);
+
+        this.userNameRef.current.value = "";
+        this.userLastNameRef.current.value = "";
+        }
+    };
+
+    onReset = () => {
+        this.userNameRef.current.value = "";
+        this.userLastNameRef.current.value = "";
+        // todo: имплементнуть функцию скидывания значений, введеных пользователем
+    };
+
+    // onAlternativeSubmit = (event) => {
+    //     event.preventDefault();
+    //     const {addUser} = this.props;
+    //
+    //     const formData = new FormData(event.target);
+    //     debugger
+    //     const user = {
+    //         name: formData.get('userName'),
+    //         lastName: formData.get('userLastName'),
+    //         id: uniqId()
+    //     };
+    //     addUser(user);
+    //
+    //     console.log(user)
+    // };
+
+    render() {
+        return (
+            <form className={CN} onSubmit={this.onSubmit}>
+                <h2>Create new User</h2>
+                <div className="form-group">
+                    <label htmlFor="title" className="input-group-text">Enter post title:</label>
+                    <input
+                        ref={this.userNameRef}
+                        className={`form-control ${this.state.errorUserName  ? "error-input" : ""}`}
+                        type="text"
+                        name="userName"
+                        onChange={this.onInputChange}
+                    />
+                <div className={'error'}> {this.userNameRef.current ? this.state.errorUserName : ""}</div>
+                </div>
+                <div className="form-group">
+                    <label htmlFor="text" className="input-group-text">Enter post text:</label>
+                    <input
+                        ref={this.userLastNameRef}
+                        className={`form-control ${this.state.errorUserLastName ? "error-input": ""}`}
+                        type="text"
+                        name="userLastName"
+                        onChange={this.onInputChange}
+                    />
+                    <div className={'error'}> {this.userLastNameRef.current ? this.state.errorUserLastName : ""}</div>
+                </div>
+                <div className={'error h3'}>
+                    {this.state.currentError ? <img className={'img-warning'} src={ImgWarning} alt=""/> : ""}
+                    <br/>
+                    {this.state.currentError}
+                </div>
```

Comment:
You can make it a more simple

`
{
!!this.state.currentError && (
<div className={'error h3'}>
<img className={'img-warning'} src={ImgWarning} alt=""/>
<br/>
                    {this.state.currentError}
                </div>
)
}
`